README.md

```diff
@@ -1,72 +1,44 @@
-   ```bash
-   make build
-   ```
-
-3. **Run the demo with instrumentation:**
-   ```bash
-   make demo
-   ```
-
-   This will:
-   - Build the instrumentation tool (`otel` binary)
-   - Run the demo with compile-time instrumentation
-   - Execute the instrumented binary
-
-## How It Works
-
-The instrumentation tool injects OpenTelemetry code at compile time using Go's `-toolexec` flag. The process works as follows:
+This project provides a tool to automatically instrument Go applications with [OpenTelemetry](https://opentelemetry.io/) at compile time. The tool modifies the Go build process to inject OpenTelemetry code into the application without requiring manual changes to the source code.
 
-1. **Setup Phase**: Creates `.otel-build` directory and initializes logging
-2. **Go Build Phase**: Intercepts the `go build` command and adds `-toolexec=./otel` flag
-3. **Toolexec Phase**: The Go toolchain invokes the tool for each compilation step, allowing code injection. This phase is not visible to the user.
-
-### Logging
-
-The tool provides detailed logging to help debug instrumentation issues:
-- Logs are written to `.otel-build/debug.log` for setup and go actions
-- Toolexec action logs to stdout for immediate feedback
-- All logs include structured information about the build process
-
-### Inspecting the Instrumentation
-
-To see the result of instrumentation, navigate to the WORK directory that appears in the build output:
+## Getting Started
 
+1. Build the otel tool
 ```bash
-# Example WORK directory
-cd /var/folders/x9/fddsvlt5363c0plvvw8_2mr80000gn/T/go-build2020695287
-
-# Locate the main package under the b001 subdirectory
-ls -l b001
+$ git clone https://github.com/open-telemetry/opentelemetry-go-compile-instrumentation.git
+$ cd opentelemetry-go-compile-instrumentation
+$ make build
+```
 
-# Inspect the modified files
-cat modified.go
+2. Build the application with the tool and run it
+```bash
+$ make demo
 ```
 
+## Project Structure
+- `docs/`: Documentation for the project
+- `demo/`: A sample Go application to demonstrate the tool
+- `instrumentation/`: Contains the instrumentation logic for various Go packages
+- `pkg/`: The core library used by `instrumentation/`, including utilities and common functionality
+  - `pkg/inst/`: Definition of the instrumentation context
+  - `pkg/inst-api/`: The API for the instrumentation
+  - `pkg/inst-api-semconv/`: The semantic conventions for the instrumentation
+- `tool/`: The main tool for compile-time instrumentation
+  - `tool/internal/setup/`: Setup phase, it prepares the environment for future instrumentation phase
+  - `tool/internal/instrument`: Instrument phase, where the actual instrumentation happens
+  - `tool/internal/rule/`: The rule describes how to match the target function and which instrumentation to apply
+
 ## Contributing
 
-See the [contributing documentation](CONTRIBUTING.md).
+See the [contributing documentation](./docs/CONTRIBUTING.md).
```

Comment:
Let's leave CONTRIBUTING.md in the root.
I have checked other OpenTelemetry repos that seems to be the convention.